README.md

```diff
@@ -42,4 +42,85 @@ To build for macOS, run the following:
 $ cargo build --target x86_64-apple-darwin --release
 ```
 
-As with iOS, link with SystemConfiguration.framework when building your macOS app.
+rs with iOS, link with SystemConfiguration.framework when building your macOS app.
+
+
+## Android Build Instructions
+
+To build for android devices you must first download the apropriate binary
+distribution of standard library for valid target platform. This can be done using rustup:
+
+```
+rustup target add armv7-linux-androideabi
+# there are other targets that might be appropriate
+rustup show | grep android
+```
+
+To setup compilation for android you will also need to tell cargo 
+about other compilation tools especialy linker. To do so download [NDK](https://developer.android.com/ndk)
+and 
```

Comment:
And what? (Looks like something is missing.)

Reply:
fixed 👍